IEEE-PSH/hackpsh-web
# Migrate Script for Drizzle

# Description
- Branch from master
- Created a script to migrate Drizzle tables to the database.
- Makes it easier to create and migrate schemas (for prod and dev).
- Can update schemas in local Supabase instances.
- Migrating existing schemas should automatically be skipped.

# Test Guidelines

- Start local supabase server through Docker using `npx supabase start`
- Run the script with `pnpm migrate`
- Ensure the script results in a success outputted through the terminal
